test/integration/webhook/jobset_webhook_test.go

```diff
@@ -563,5 +563,10 @@ var _ = ginkgo.Describe("jobset webhook defaulting", func() {
 			},
 			updateShouldFail: true,
 		}),
+		ginkgo.Entry("ReplicatedJob missing", &testCase{
+			makeJobSet: func(ns *corev1.Namespace) *testing.JobSetWrapper {
+				return testing.MakeJobSet("depends-on", ns.Name)
+			},
```

Comment:
Should the creation fail in that test case ?
`jobSetCreationShouldFail: true,`

Reply:
maybe. this is WIP to demonstrate the failure and figure out what we should do that about.

Reply:
https://github.com/kubernetes-sigs/kueue/pull/4467#discussion_r1978521961

Reply:
If we had the failure this will pass but the failure is related to dependsOn exception.